docs/en/docs/advanced/extending-openapi.md

```diff
@@ -165,7 +165,7 @@ After that, your file structure could look like:
 
 Start your application and go to <a href="http://127.0.0.1:8000/static/redoc.standalone.js" class="external-link" target="_blank">http://127.0.0.1:8000/static/redoc.standalone.js</a>.
 
-You should see a very long JavaScript file for **ReDoc**.
+You should see a very long JavaScript files for **ReDoc**.
```

Comment:
This is wrong, original is correct.

Reply:
Correct. In-fact `very long` needs to be removed here and keep `file`.